docs/source/cli.rst

```diff
@@ -49,7 +49,41 @@ specified.
 Experiments
 -----------
 
-Subcommands to create and list experiments.
+Subcommands to manage experiments.
+
+
+Create
+------
```

Comment:
I think this needs to be a separate kind of header from `----` to render correctly in the docs, since it's under Experiments. Right now these will look like the same level of heading. You can look at the Sphinx/RST docs to see the hierarchy of headings allowed.

Reply:
It might also be possible to just make this a bulleted list instead of sections, since there's a lot of detail about each one.

Reply:
Oh! I misread the documentations. Subsections are marked with "~" 

I had initially tried the bullet route, but it looked a bit bulky.